### ISSUE TITLE
Define public constant for EDGEX_SECURITY_SECRET_STORE

### ISSUE THREAD

**Issue by michaelestrin** [edgexfoundry/go-mod-secrets#32]
Update corresponding usage in edgex-go (internal/pkg/bootstrap/handlers/secret.go) and docker-edgex-mongo (https://github.com/edgexfoundry/docker-edgex-mongo/blob/0769d9f0c8416f69ea421b9bf45ba0e6c6735a44/internal/pkg/constants.go#L20).

**Comment by lenny-intel** [comment]
@JamesKButcher , I don't think this is needed here and can be closed. Bootstrap now takes care of this.

**Comment by lenny-intel** [comment]
Bootstrap has this functionality
